hadoop-pipeline-application/src/main/java/org/molgenis/hadoop/pipeline/application/processes/StreamHandler.java

```diff
@@ -0,0 +1,26 @@
+package org.molgenis.hadoop.pipeline.application.processes;
+
+/**
+ * Class containing generic code for streaming input/output using {@link Runnable}.
+ */
+public abstract class StreamHandler implements Runnable
+{
+	/**
+	 * A new thread to execute the stream with.
+	 */
+	protected final Thread thread = new Thread(this);
+
+	/**
+	 * Initiates the stream writing (by calling {@code run()}).
+	 */
+	public void start()
+	{
+		thread.start();
+	}
+
+	@Override
+	public void run()
+	{
+	}
```

Comment:
Removed code instead (`run()` with `@Override` will be used when there's actual code to be placed within `run()`). Same goes for the `InStreamHandler` which will be an `abstract class` in the future push.